ocp_resources/performance_profile.py

```diff
@@ -0,0 +1,112 @@
+from ocp_resources.resource import Resource
+
+
+class PerformanceProfile(Resource):
+    api_group = Resource.ApiGroup.PERFORMANCE_OPENSHIFT_IO
+
+    def __init__(
+        self,
+        additional_kernel_args=None,
+        cpu=None,
+        globally_disable_irq_load_balancing=False,
+        hugepages=None,
+        machine_config_label=None,
+        machine_config_pool_selector=None,
+        net=None,
+        node_selector=None,
+        numa=None,
+        real_time_kernel=None,
+        workload_hints=None,
+        **kwargs,
+    ):
+        """
+        PerformanceProfile object. API reference:
+        https://docs.openshift.com/container-platform/4.14/rest_api/node_apis/performanceprofile-performance-openshift-io-v2.html#performanceprofile-performance-openshift-io-v2
+
+        Args:
```

Comment:
From the API page I see that only `cpu` and `nodeSelector` are Required
And for `spec.cpu` `isolated` and `reserved` are Required

Please update the docstring and the code.

Reply:
done